token-lending/program/src/state/reserve.rs

```diff
@@ -617,739 +700,198 @@ impl Pack for Reserve {
         let (
             version,
```

Comment:
maybe we can start splitting this up by reserve, liquidity, collateral, config, etc

Reply:
I agree. Obligation/Liquidity/Collateral has the same issue. I was thinking about it but didn't want to keep expanding this PR. I'll add a TODO though.

Reply:
It should be minor enough to add here, right?